flink-python/setup.py

```diff
@@ -75,8 +85,9 @@
             print("Temp path for symlink to parent already exists {0}".format(TEMP_PATH),
                   file=sys.stderr)
             sys.exit(-1)
-
-        FLINK_HOME = os.path.abspath("../build-target")
+        java_version = VERSION.replace(".dev0", "-SNAPSHOT")
```

Comment:
How about renaming it to flink_version? I was first confused that it is the version of jvm.